game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyLocationFileDownloader.java

```diff
@@ -1,17 +1,15 @@
 package games.strategy.engine.lobby.client.login;
 
-import com.google.common.annotations.VisibleForTesting;
+import java.io.File;
+import java.util.Optional;
 
-import games.strategy.engine.framework.map.download.DownloadUtils;
+import com.google.common.annotations.VisibleForTesting;
 
 /**
  * Humble object pattern. Wraps file downloading.
  */
 @FunctionalInterface
 @VisibleForTesting
 interface LobbyLocationFileDownloader {
-
-  DownloadUtils.FileDownloadResult download(final String url);
-
-  LobbyLocationFileDownloader defaultDownloader = DownloadUtils::downloadToFile;
+  Optional<File> download(final String url);
```

Comment:
Maybe that's just me, but in my opinion it would be sufficient to use a `Function<String, Optional<File>>` instead. This interface is only used internally anyways.

Reply:
That's a good point that, with the removal of the field, this interface has simply degenerated to `Function<String, Optional<File>>`.  However, I'm going to merge this now to help empty the PR queue.  I'll submit a PR later to implement this, and @DanVanAtta can reject it if he feels there's still a need to have a dedicated interface.